documentation-website/src/pages/usage/routes/index.tsx

```diff
@@ -22,7 +22,7 @@ const Route = () => <Layout
       </p>
       <Code language='typescript'>{`import {
   Task,
-} from '@idrinth/api-bench';
+} from '@idrinth-api-bench/api-bench';
```

Comment:
this package will never exist, please revert the change for now

Reply:
Reverted